### ISSUE TITLE
glTF support status

### ISSUE THREAD

**Issue by micronAMZN** [o3de/o3de#6007]
Support for glTF in O3DE seems more dependent on the varying DCC export implementations at this time. After testing glTF support with various DCC applications these are the results:

**Formats** - O3DE can process source assets stored in:
* `.gltf`
* `.glb`
* `.gltf` + `.bin`

**Meshes** - Mesh import works from every DCC spplication I surveyed with support for the following features:

* LOD - Meshes are properly supported with the `_lod{0-4}` suffix.
* PhysX - Meshes are properly supported with the `_phys` suffix.
* UVs - Multiple UV sets are supported.
* Vertex Colors - Multiple color sets are supported.
* Normals - Custom normals are supported.
* Tangents - Custom tangents are supported.

**Materials**

tldr; Users have to set up their own materials in O3DE.

Materials are properly applied in O3DE for all three formats. Automatic texture assignment only works when the source asset is `.gltf` + `.bin` and the textures are not embedded in the glTF asset. Although the glTF asset contains the correct material property values, they are not applied to the material in O3DE when processed.

**Actors**

There currently seem to be very limited options for writing skeleton information to glTF. I had the best success with Blender. In contrast to FBX, the gltf skeleton did not contain extraneous nodes. Mesh skin weights and bone indices seem to be properly supported but more testing is needed. 

**Motions**

Again I had the best success with Blender. Can successfully process a motion from a Blender generated glTF. More testing is needed for multiple motions. Might be possible to store the Actor in glTF and process motion from FBX.

**PhysX**

PhysX meshes with the `_phys` suffix are properly supported. Physics materials need more testing.

**Untested features**

* Cloth
* Physics materials
* Blast - there are currently larger issues with Blast, so this is not expected to work.

**Issues**

* In general, support for animated actors on the DCC side seems to be very poor. Blender was the only application that proved successful.
* Though Blender seemed to offer the most complete support, there were significant issues with scene scale, Assets in O3DE were microscopic no matter how the scene units in Blender were set. This might be user error, more testing is needed.
*  The issues with materials seem to be on the O3DE/assimp side. The material properties are properly recorded in glTF.

**Comment by AMZN-stankowi** [comment]
This improvement helps GLTF files coming it at the wrong scale https://github.com/o3de/o3de/pull/12112
